src/fsharp/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -30,9 +30,30 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
   </Choose>
 
   <PropertyGroup>
-    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
     <EnableDefaultCompileItems Condition=" '$(EnableDefaultCompileItems)' == '' ">false</EnableDefaultCompileItems>                                <!--- Do not glob F# source files -->
+    <EnableDefaultNoneItems Condition=" '$(EnableDefaultNoneItems)' == '' ">false</EnableDefaultNoneItems>
+  </PropertyGroup>
+
+  <PropertyGroup>
     <DefaultProjectTypeGuid Condition=" '$(DefaultProjectTypeGuid)' == '' ">{F2A71F9B-5D33-465A-A702-920D77279786}</DefaultProjectTypeGuid>         <!-- F# project type -->
+    <DocumentationFile>bin\$(Configuration)\$(AssemblyName).XML</DocumentationFile>
+    <Prefer32Bit>false</Prefer32Bit>
+    <Tailcalls>true</Tailcalls>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
+    <WarningLevel Condition=" '$(WarningLevel)' == '' ">3</WarningLevel>
+    <WarningsAsErrors />
+    <DebugSymbols>true</DebugSymbols>
+    <DebugType>portable</DebugType>
+  </PropertyGroup>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <Optimize>false</Optimize>
+    <DefineConstants>$(DefineConstants);DEBUG</DefineConstants>
```

Comment:
I would remove DefineConstants as well - the SDK adds a constant for the TFM. I think if we do this then the code in the SDK may not add the TFM constant since it's already defined here.